ari.lua

```diff
@@ -348,11 +349,23 @@ function ari.dissector(buffer, pinfo, tree)
     header_tree:add(pkt_trx, buffer(10, 2), pkt_transaction, "Transaction: " .. pkt_transaction.. " (" .. string.format("0x%08x", pkt_transaction) .. ")")
     packet.trx = pkt_trx
 
+    --- Acknowledgement Option
+    local ack_opt = buffer(8, 1):bitfield(4, 1)
+
+    header_tree:add(pkt_ack_opt, buffer(8, 1), ack_opt, "Acknowledgement Option: " .. ack_opt)
+    packet.ack_opt = ack_opt
+
+    --- Virtual field GMID
+    -- the "gmid" is a virtual id consisting of the type (or "mid") and group ("gid")
+    local gmid = bit32.bor(bit32.lshift(pkt_group_int, 26), bit32.lshift(pkt_type_int, 15))
+
+    header_tree:add(pkt_gmid, buffer(8, 2), gmid, "Gm id (Group + Type): " .. gmid .. " (" .. string.format("0x%08x", gmid) .. ")")
```

Comment:
```suggestion
    header_tree:add(pkt_gmid, buffer(4, 6), gmid, "Gm id (Group + Type): " .. gmid .. " (" .. string.format("0x%08x", gmid) .. ")")
```

Reply:
Fixed, but for consistency: the group field does not respect this convention either (haven't checked other fields). Should I open an issue to fix this?

Reply:
Yes please do so :)